src/templates/iframe.html.j2

```diff
@@ -239,14 +248,22 @@ function init(scrollwheel) {
             }
         }
 
+        var marker = new google.maps.Marker({
+            position: new google.maps.LatLng(sites[site][0], sites[site][1]),
+            map: map,
+            zIndex: resourceCounter,
+            icon: icon,
```

Comment:
Is "icon" defined at this point?

Reply:
Good catch.  For some reason, it didn't complain.  Fixing.